lib/node-axosoft.js

```diff
@@ -2,12 +2,9 @@ var pkg = require('../package.json');
 
 function NodeAxosoft (base_url, credentials) {
 
-	var api = require('./api');
-
-	api.register(base_url, credentials);
-
 	var node_axosoft = {
 		VERSION: pkg.version,
+		Api: require('./api'),
```

Comment:
keeping as it's needed for the access token methods and if we were to make rest calls that don't have methods built into the wrapper